packages/core/src/storage.ts

```diff
@@ -28,22 +29,99 @@ export const defaultClientStorageSetter: StorageSetter = (key, value) => {
   }
 }
 
-export const cookieStorageGetter: StorageGetter = (key) => {
-  if (isBrowser) {
-    return Cookies.get(key) ?? null
-  } else {
-    return null
+const checkStorageAccessors = (
+  clientStorage: ClientStorage,
+  accessors: Array<keyof ClientStorage>
+) => {
+  accessors.forEach((key) => {
+    if (typeof clientStorage[key] !== 'function') {
+      console.error(`clientStorage.${key} is not a function`)
+    }
+  })
+}
+
+export const localStorageGetter = (
+  clientStorageType: ClientStorageType,
+  clientStorage?: ClientStorage
+): StorageGetter => {
+  if (clientStorage) {
+    if (clientStorageType === 'react-native') {
+      checkStorageAccessors(clientStorage, ['getItem'])
```

Comment:
wouldn't it make sense to only execute this function if we are in a debug mode or something?

Reply:
yes. legacy code. Will comment it for now on